### DISCUSSION TITLE
/zh-cn/blog/posts/overview_with_colors

### DISCUSSION THREAD

**Discussion by giscus** [Reverier-Xu/discussions#145] [Announcements]
# 色彩系统与视觉设计 - Reverier's Blog

起源 光，一种用来拯救世界的信仰 通常指的是人类眼睛可见的电磁波，视知觉 就是对于可见光的知觉。 可见光只是电磁波谱上介于 400nm - 700nm 之间的部分（也有一些

https://blog.woooo.tech/posts/overview_with_colors/

**Comment by snowolf46** [comment]
啊？

**Reply by Reverier-Xu** [reply]
啊？
